app/services/engine_service.rb

```diff
@@ -0,0 +1,26 @@
+class EngineService
+  def get_url(path)
+    Faraday.get("http://localhost:3000/api/v1/#{path}")
+  end
+
+  def all_merchants
+    response = get_url("/merchants")
+    data = JSON.parse(response.body, symbolize_names: true)
+
+    data[:data]
+  end
+
+  def merchant(id)
+    response = get_url("/merchants/#{id}")
+    data = JSON.parse(response.body, symbolize_names: true)
+
+    data[:data]
+  end
+
+  def merchant_items(id)
+    response = get_url("/merchants/#{id}/items")
+    data = JSON.parse(response.body, symbolize_names: true)
+
+    data[:data]
+  end
```

Comment:
on lines 10, 17, and 24, you are going into the data root key. Let your facade be responsible for filtering through the data that is returned, your service should just be responsible for making a call, parsing the data and sending it back. But there may be other root keys outside of the data key that your facade may eventually want -- dont limit what the facade can have right away.